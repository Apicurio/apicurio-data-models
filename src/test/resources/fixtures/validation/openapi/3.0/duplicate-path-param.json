{
  "openapi": "3.0.2",
  "info": {
    "version": "1.0.0",
    "title": "Swagger Petstore"
  },
  "paths": {
    "/dupesWithinPathItemInlineAndRefCombo": {
      "get": {
        "description": "Duplicate parameters (name & in combination) one defined inline within operation and other referenced - invalid",
        "responses": {
          "default": {
            "description": "..."
          }
        }
      },
      "parameters": [
        {
          "name": "status",
          "in": "query",
          "description": "Status param in query",
          "required": false,
          "schema": {
            "type": "string"
          }
        },
        {
          "$ref": "#/components/parameters/statusParam"
        }
      ]
    }
  },
  "components": {
    "parameters": {
      "statusParam": {
        "name": "status",
        "in": "query",
        "description": "Status param in query",
        "required": false,
        "schema": {
          "type": "string"
        }
      },
      "indirectRef": {
        "$ref": "#/components/parameters/statusParam"
      }
    }
  }
}
