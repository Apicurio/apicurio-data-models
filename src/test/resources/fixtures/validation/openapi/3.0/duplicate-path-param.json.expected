[PAR-019] |medium| {/paths[/dupesWithinPathItemInlineAndRefCombo]/parameters[0]->in} :: Duplicate query parameter named 'status' found (parameters must be unique by name and location).
[PAR-019] |medium| {/paths[/dupesWithinPathItemInlineAndRefCombo]/parameters[1]->in} :: Duplicate query parameter named 'status' found (parameters must be unique by name and location).